Mathlib/Analysis/LocallyConvex/Bounded.lean

```diff
@@ -240,17 +242,67 @@ theorem isVonNBounded_singleton (x : E) : IsVonNBounded 𝕜 ({x} : Set E) := fu
   (absorbent_nhds_zero hV).absorbs
 #align bornology.is_vonN_bounded_singleton Bornology.isVonNBounded_singleton
 
-protected theorem IsVonNBounded.vadd [ContinuousAdd E] {s : Set E}
-    (hs : IsVonNBounded 𝕜 s) (x : E) : IsVonNBounded 𝕜 (x +ᵥ s) := by
+section ContinuousAdd
+
+variable [ContinuousAdd E] {s t : Set E}
+
+protected theorem IsVonNBounded.vadd (hs : IsVonNBounded 𝕜 s) (x : E) :
+    IsVonNBounded 𝕜 (x +ᵥ s) := by
   rw [← singleton_vadd]
   -- TODO: dot notation timeouts in the next line
   exact IsVonNBounded.add (isVonNBounded_singleton x) hs
 
 @[simp]
-theorem isVonNBounded_vadd [ContinuousAdd E] {s : Set E} (x : E) :
-    IsVonNBounded 𝕜 (x +ᵥ s) ↔ IsVonNBounded 𝕜 s :=
+theorem isVonNBounded_vadd (x : E) : IsVonNBounded 𝕜 (x +ᵥ s) ↔ IsVonNBounded 𝕜 s :=
   ⟨fun h ↦ by simpa using h.vadd (-x), fun h ↦ h.vadd x⟩
 
+theorem IsVonNBounded.right_of_add (hst : IsVonNBounded 𝕜 (s + t)) (hs : s.Nonempty) :
```

Comment:
```suggestion
theorem IsVonNBounded.of_add_right (hst : IsVonNBounded 𝕜 (s + t)) (hs : s.Nonempty) :
```

Reply:
With `of_add_right`, it's not clear (for me) whether it's "of adding something on the right" (so this one should be `of_add_left`, see `Int.ModEq.of_mul_left`) or "of add, version for right term".

Reply:
That's fine, because both interpretations mean the same thing. We already used this pattern in other places.

Reply:
Should this lemma be `of_add_left` or `of_add_right`?

Reply:
I also used `left_of_*` in, e.g., `left_ne_zero_of_mul`. I don't care about this case (e.g., feel free to push a change to this PR) but we should choose one style some day. For now, for all lemmas ending with `left`/`right`, I try both every time.

Reply:
The difference between `left_ne_zero_of_mul` and here is that we have access to dot notation here, which means that we flush the `left`/`right` to the end of the lemma name.